src/gwt/src/org/rstudio/studio/client/workbench/views/output/lint/LintRetina.css

```diff
@@ -10,6 +11,10 @@
    background-image: WARNING;
 }
 
+.spelling {
+   background-image: SPELLING;
```

Comment:
for now the image is the same as WARNING, but it could be interesting to style spelling type differently.